pkg/controller/configmapcainjector/starter.go

```diff
@@ -0,0 +1,59 @@
+package configmapcainjector
+
+import (
+	"fmt"
+	"time"
+
+	"k8s.io/client-go/kubernetes"
+	"k8s.io/client-go/rest"
+
+	"crypto/x509"
+	"encoding/pem"
+	"io/ioutil"
+
+	"github.com/golang/glog"
+	servicecertsignerv1alpha1 "github.com/openshift/api/servicecertsigner/v1alpha1"
+	"k8s.io/client-go/informers"
+)
+
+type ConfigMapCABundleInjectorOptions struct {
+	Config *servicecertsignerv1alpha1.ConfigMapCABundleInjectorConfig
+}
+
+func (o *ConfigMapCABundleInjectorOptions) RunConfigMapCABundleInjector(clientConfig *rest.Config, stopCh <-chan struct{}) error {
+	kubeClient, err := kubernetes.NewForConfig(clientConfig)
+	if err != nil {
+		return err
+	}
+	kubeInformers := informers.NewSharedInformerFactory(kubeClient, 2*time.Minute)
+
+	caPem, err := ioutil.ReadFile(o.Config.CABundleFile)
```

Comment:
If the operator deployment requires you to mount in the bundle, what's the point in making the file configurable as opposed to a well-known location?

Reply:
It's part of the OpenShift API for the operator https://github.com/openshift/api/blob/master/servicecertsigner/v1alpha1/types.go#L62